app/services/contacts_service.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class ContactsService
+  def by_content_id(content_id)
+    # FIXME: This might return a draft as this Publishing API method doesn't
+    # distinguish between these
+    document = GdsApi.publishing_api_v2.get_content(content_id)
+    document["schema_name"] == "contact" ? document.to_h : nil
```

Comment:
You probably thought of this already, but it looks like the response says which content store it's coming from. Could we add that filter in here as well?

Reply:
Unfortunately that'd remove things that are live and have a draft